build/debs/build.go

```diff
@@ -20,41 +21,54 @@ type DistributionType string
 
 const (
 	DistributionStable   DistributionType = "stable"
-	DistributionUnstable DistributionType = "unstable"
 	DistributionTesting  DistributionType = "testing"
+	DistributionUnstable DistributionType = "unstable"
+
+	minimumKubernetesVersion = "1.13.0"
+	minimumCNIVersion        = "0.7.5"
+	pre117CNIVersion         = "0.7.5"
 
-	cniVersion      = "0.7.5"
-	criToolsVersion = "1.13.0"
+	defaultRevision = "0"
 
 	packagesRootDir = "packages"
 
 	kubeadmConf = "10-kubeadm.conf"
 )
 
-var latestPackagesDir = fmt.Sprintf("%s/%s", packagesRootDir, "latest")
+var (
+	minimumCRIToolsVersion = minimumKubernetesVersion
```

Comment:
I'm not sure this a safe default.  It's probably the general intent, but we regularly end up with mismatches because things weren't tested in conjunction with one another and then move forward out of sync, eg: https://github.com/kubernetes/kubernetes/pull/83217

Reply:
I think the default should be to build a specified combination.  These dependencies should be specified elsewhere consistently and everything consume that source of truth, instead of making other default assumptions.